src/App.jsx

```diff
@@ -1,18 +1,17 @@
 import "./App.css";
-import Homepage from "./components/Homepage";
-import { Routes, Route, useNavigate } from "react-router-dom";
+import { Link, Outlet } from "react-router-dom";
 
-function App() {
+export default function App() {
```

Comment:
Let's consider `App` as a frame of our application. It includes navigation (`.App__nav`), and container for page content (`.App__main`)